airbyte-integrations/connectors/source-gitlab/Dockerfile

```diff
@@ -4,8 +4,8 @@ FROM python:3.9-slim
 RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
-#COPY --from=ghcr.io/estuary/airbyte-to-flow:dev /airbyte-to-flow ./
-#
+COPY --from=ghcr.io/estuary/airbyte-to-flow:dev /airbyte-to-flow ./
```

Comment:
nit: Maybe move this down to line 16 so the pattern is consistent across all these Dockerfiles?